### PR TITLE
Grammatical and formatting fixes

### PR DESCRIPTION
**Pull request by Hamled**
Added periods in places where it made sense. In English periods are more of a statement terminator than a statement separator -- ask your ESL teacher.
Also changed "advantaged" to "advantage" in the Portal project description.
Also fixed some formatting around the task lists for the Portal and Titan projects.

### CODE REVIEW

**Comment by tomjakubowski** [comment]
lol

**Comment by Ell** [comment]
I need a visual dif of the changes and tests before I can accept this PR. 

Looks good so far though, keep up the good work.

File: index.html [hunk 1]
```diff
@@ -123,22 +124,22 @@
 
                     <div class="exp">
                         <h4>&raquo; Lead Developer at NewComLink (2014 - Now)</h4>
-                        <p>Took lead of a small team (around 5 developers) to create a new and modern user experience for the companies main product. This included creating a well tested and documented AngularJS application that communicated with a RESTful API created in house</p>
+                        <p>Took lead of a small team (around 5 developers) to create a new and modern user experience for the companies main product. This included creating a well tested and documented AngularJS application that communicated with a RESTful API created in house.</p>
```
Comment (tomjakubowski) [inline index.html]: in-house

Comment (tomjakubowski) [inline index.html]: company's

File: index.html [hunk 2]
```diff
@@ -123,22 +124,22 @@
...
                     </div>
 
                     <div class="exp">
                         <h4>&raquo; Senior Developer at The University of Arizona (2012 - 2014)</h4>
-                        <p>Created and maintained websites deployed university wide. Coded mainly using Django with some Flask/SQLAlchemy. Supported and updated other projects coded in a variety of other frameworks and languages</p>
+                        <p>Created and maintained websites deployed university wide. Coded mainly using Django with some Flask/SQLAlchemy. Supported and updated other projects coded in a variety of other frameworks and languages.</p>
```
Comment (tomjakubowski) [inline index.html]: "university-wide" or "across the university". It's unclear whether "coded" is an adjective (describing the websites) or a verb. "Coded" is bad either way.

My suggestion:

> Created and maintained websites for the university, written in Python using Django, Flask, and SQLAlchemy. Supported other projects written in $language_keywords and $framework_keywords.

**Comment by tomjakubowski** [comment]
all but one of your experience section paragraphs starts with "Took" or "Created". spice up your life

File: index.html [hunk 3]
```diff
@@ -150,10 +151,12 @@
...
                         <h4>&raquo; Portal (AngularJS / Node)</h4>
-                        <p>Created and managed an AngularJS project that took advantaged of our internal API to provide a modern front end experience for our users
-
+                        <p>Created and managed an AngularJS project that took advantage of our internal API to provide a modern front end experience for our users.
+                        
+                        <br/>
                         This includes:
-
+                        <br/>
+                                                                                                                                                                                                                                                                                                    <!-- I ask other people to fix my resume. Think of what I will do with my^H^Hyour code. -->
```
Comment (tomjakubowski) [inline index.html]: lmao

File: index.html [hunk 4]
```diff
@@ -123,22 +124,22 @@
...
                     <div class="exp">
                         <h4>&raquo; Lead Developer at Ascension Marketing (2012)</h4>
-                        <p>Created a custom homepage solution based around the Flask framework that incorporated many API's and services including Shopify and Facebook</p>
+                        <p>Created a custom homepage solution based around the Flask framework that incorporated many API's and services including Shopify and Facebook.</p>
                     </div>
 
                     <div class="exp">
                         <h4>&raquo; Web Design at The University of Arizona (2010)</h4>
-                        <p>Integrated designs from various PDF and PSD's to create a HTML template for use in a CMS</p>
+                        <p>Integrated designs from various PDF and PSD's to create a HTML template for use in a CMS.</p>
```
Comment (tomjakubowski) [inline index.html]: PDFs and PSDs

File: index.html [hunk 5]
```diff
@@ -123,22 +124,22 @@
...
                     <div class="exp">
                         <h4>&raquo; Senior Developer at The University of Arizona (2012 - 2014)</h4>
-                        <p>Created and maintained websites deployed university wide. Coded mainly using Django with some Flask/SQLAlchemy. Supported and updated other projects coded in a variety of other frameworks and languages</p>
+                        <p>Created and maintained websites deployed university wide. Coded mainly using Django with some Flask/SQLAlchemy. Supported and updated other projects coded in a variety of other frameworks and languages.</p>
                     </div>
 
                     <div class="exp">
                         <h4>&raquo; Lead Developer at Ascension Marketing (2012)</h4>
-                        <p>Created a custom homepage solution based around the Flask framework that incorporated many API's and services including Shopify and Facebook</p>
+                        <p>Created a custom homepage solution based around the Flask framework that incorporated many API's and services including Shopify and Facebook.</p>
```
Comment (tomjakubowski) [inline index.html]: APIs

File: index.html [hunk 6]
```diff
@@ -150,10 +151,12 @@
...
-                        <p>Created and managed an AngularJS project that took advantaged of our internal API to provide a modern front end experience for our users
-
+                        <p>Created and managed an AngularJS project that took advantage of our internal API to provide a modern front end experience for our users.
+                        
+                        <br/>
                         This includes:
-
+                        <br/>
+                                                                                                                                                                                                                                                                                                    <!-- I ask other people to fix my resume. Think of what I will do with my^H^Hyour code. -->
                         <ul>
```
Comment (tomjakubowski) [inline index.html]: lmao

**Comment by sailorswift** [comment]
:thumbsup::thumbsup::thumbsup:

**Comment by rmmh** [comment]
:+1:

File: index.html [hunk 7]
```diff
@@ -150,10 +151,12 @@
 
                     <div class="exp">
                         <h4>&raquo; Portal (AngularJS / Node)</h4>
-                        <p>Created and managed an AngularJS project that took advantaged of our internal API to provide a modern front end experience for our users
-
+                        <p>Created and managed an AngularJS project that took advantage of our internal API to provide a modern front end experience for our users.
+                        
+                        <br/>
```
Comment (srh) [inline index.html]: <br/>?  This is HTML, that solidus is a waste of bandwidth.

**Comment by seubert** [comment]
:+1: :100: :8ball: :point_up_2:

**Pull request merged**